files/shorewall

```diff
@@ -0,0 +1,5 @@
+SHOREWALL_TCP .*?%{WORD:fwrule}\:%{WORD:fwtarget}\:IN\=%{WORD:fwin} OUT\=\s*MAC\=%{DATA:fwmac} SRC\=%{IPORHOST:fwsrc} DST\=%{IPORHOST:fwdst} .*? PROTO\=%{WORD:fwproto} SPT\=%{INT:fwspt:int} DPT\=%{INT:fwdpt:int} .*?
```

Comment:
@wpoely86 yes but logstash MAC type only match those cases:
`MAC (?:%{CISCOMAC}|%{WINDOWSMAC}|%{COMMONMAC})`
In some cases it does not work with some MACs provided by shorewall logs...